### ISSUE TITLE
Search responder queries are not deterministic

### ISSUE THREAD

**Issue by benjamingeer** [dhlab-basel/Knora#11]
Two problems:

1. In a full-text search (e.g. search for 'Orationes' in Incunabula), if a resource has a matching label and a matching value, Fuseki and GraphDB return different results: Fuseki returns the match for the resource label, while GraphDB returns the one for the text value. Both appear to be correct: we are using SAMPLE, so each triplestore is returning a different random result. Try to find another approach so that they return the same result.
2. GraphDB returns ontology entity labels in the wrong language (again, this seems to be because we have erroneously requested a random one, rather than the one in the user's preferred language as intended). A good solution would probably be to remove all ontology-related things from the search queries, and instead to have the search responder ask the ontology responder for that information.

**Comment by benjamingeer** [comment]
`wip/ontology-in-search`

**Comment by tobiasschweizer** [comment]
I have just thought about the handling of valueHasComment in the search templates #22

**Comment by tobiasschweizer** [comment]
Summarizing the changes you made, I mention the following steps:
- in the SPARQL templates for fulltext and extended search, the language dependent labels for resource classes were removed, instead we will now deal with the Iris of the resource class and the property type returned by the query
- the SearchResultsRowV1s are wrapped in another case class called SearchResultRowWithEntityIris / SearchResultRowWithResourceClassIri exposing the resource class IRI (and the property type Iri in case of a fulltext search)
- these Iris are passed to the ontology responder
- the original SearchResultRowV1s are copied and completed with the information returned by the ontology responder

I have the following comments:

- in the search template, a `resourceLabel` is returned containing the `rdfs:label` of the resource found. This is absolutely fine in the RDF-terminology, but it is easily confused with a language dependent label like a "Ein Buch". Fot this reason, I found firstProp clearer (but this is some kind of SALSAH legacy).

-  in the fulltext template, there is a variable called `?valueObjectResource`. This is the resource relating to the found value object via a property. The name is not very clear to me.

- now, we have two steps: first the query is done and the results are put in a bunch of `SearchResultsRowV1`, which are wrapped into `SearchResultRowWithEntityIris` / `SearchResultRowWithResourceClassIri `. When the ontology responder has been called, the missing information are added to `SearchResultsRowV1` (the case class is copied). Could this be done without creating `SearchResultsRowV1` before calling the ontology responder?
